src/__internal__/character-count/character-count.component.tsx

```diff
@@ -9,19 +10,47 @@ interface CharacterCountProps {
   "data-element"?: string;
 }
 
+export const PolishPlural = (
```

Comment:
I feel this should be in it's own file separate from the component. I'm kind of worried that if we've done this for Polish, what other languages would we be expected to have a need to write a function for also?